### DISCUSSION TITLE
Connection Failed

### DISCUSSION THREAD

**Discussion by MaAr2009** [ORelio/Minecraft-Console-Client#2773] [Q&A] [answered]
The connection has not been established because The destination computer rejected the connection request. [::ffff:88.888.888.88]:25565
Failed to connect to this IP.

Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение.  [::ffff:88.888.888.88]:25565
Не удалось подключиться к этому IP.

**Comment by milutinke** [comment, marked as answer]
The MCC doesn't support IPv6
